GiEnJul/ClientApp/src/routes/registerGiver/SignUp.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import { render } from 'react-dom';
+import Confirmation from './Confirmation';
+import ContactInfo from './ContactInfo';
+import LocationGiver from './LocationGiver';
+import SummaryRegistration from './SummaryRegistration';
+
+interface State {
+    step: number,
+    location: string | undefined,
+    name: string | undefined,
+    email: string | undefined,
+    tlf: number | undefined,
+    familiyType: string | undefined,
```

Comment:
These could probably be shortened to e.g. `location?: string` instead of `location: string | undefined`.